opt/opt.rs

```diff
@@ -125,6 +125,10 @@ pub struct Dominators {
     /// The format the output should be written in.
     #[structopt(short = "f", long = "format", default_value = "text")]
     pub output_format: OutputFormat,
+
+    #[structopt(short = "d", default_value = "10")] pub max_depth: usize,
+
+    #[structopt(short = "r", default_value = "10")] pub max_row: usize,
```

Comment:
NItpick: `max_rows` instead of `max_row.

Let's also add doc comments to each new field, since this will turn into the CLI help text:

> The maximum depth to print the dominators tree.

> The maximum number of rows, regardless of depth in the tree, to display.

Let's also make these `Option<usize>` rather than providing a default value.